src/index.html

```diff
@@ -15,8 +15,106 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
```

Comment:
Lines of code have `80` chars max. Consider breaking the `href` attribute value into multiple lines to adhere to the maximum line length.